packages/core/src/App/Containers/Layout/header/dtrader-header.tsx

```diff
@@ -0,0 +1,140 @@
+import classNames from 'classnames';
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { routes, getDecimalPlaces, platforms } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+// import { useWalletMigration } from '@deriv/hooks';
```

Comment:
Please check this commented code

Reply:
@nijil-deriv these commented codes are removed from the file after confirming with @aum-deriv that they are from wallets v1 and are not required in the future. please let me know if you have any concerns 🙏🏼